content/kotlin/concepts/strings/terms/lowercase/lowercase.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: '.lowercase()'
+Description: 'The lowercase method in Kotlin is used to convert the characters of a string to lowercase.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The `.lowercase()` method in Kotlin is used to convert the characters of a string to lowercase. 
+
+## Syntax
+
+```pseudo
+fun String.lowercase(locale: Locale = Locale.getDefault()): String
+```
+
+- `locale` (optional): A `Locale` object representing the language and region-specific rules for case conversion. The default value is `Locale.getDefault()`.
+
+## Examples
+
+Let's dive into some examples to see how the `.lowercase()` method can be used:
+
+```kotlin
+fun main() {
+    println("Iced frappé!".lowercase())
+}
+```
+
+The output of this code will be:
+
+```shell
+iced frappé!
+```
+
+Let's take a look at an example where we specify the locale:
+
+```kotlin
+import java.util.Locale
```

Comment:
This example demonstrates the use of the optional parameter, but it is not a case in which setting the locale results in a change. It would be better to use a case in which there was a different output.